source/agora/utils/PrettyPrinter.d

```diff
@@ -275,15 +283,17 @@ private struct TransactionFmt
             enum OutputPerLine = 3;
 
             if (this.value.inputs.length)
-                formattedWrite(sink, "Type : %s, Inputs (%d): %(%(%s, %),\n%)\n",
+                formattedWrite(sink, "Type : %s, Inputs (%d):%s%(%(%s, %),\n%)\n",
                     this.value.type,
                     this.value.inputs.length,
+                    startNewLineIfMultiple(InputPerLine, this.value.inputs.length),
```

Comment:
`this.value.inputs.length > InputPerLine ? "\n" : " "`

Reply:
I was thinking ternary operator was not available in D which is why I created the function. Will update.

Reply:
Updated